app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyHomeDetailScreen.kt

```diff
@@ -0,0 +1,174 @@
+package com.kks.nimblesurveyjetpackcompose.ui.presentation.survey
+
+import androidx.compose.animation.Crossfade
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+import com.google.accompanist.pager.ExperimentalPagerApi
+import com.google.accompanist.pager.HorizontalPager
+import com.kks.nimblesurveyjetpackcompose.R
+import com.kks.nimblesurveyjetpackcompose.model.Survey
+import com.kks.nimblesurveyjetpackcompose.ui.theme.BlackRussian
+import com.kks.nimblesurveyjetpackcompose.ui.theme.NeuzeitFamily
+import com.kks.nimblesurveyjetpackcompose.ui.theme.White70
+import com.kks.nimblesurveyjetpackcompose.util.TWEEN_ANIM_TIME
+import com.ramcosta.composedestinations.annotation.Destination
+
+@OptIn(ExperimentalPagerApi::class)
+@Destination
+@Composable
+fun SurveyHomeDetailScreen(survey: Survey) {
+    var currentPage by remember { mutableStateOf(0) }
+
+    Box(modifier = Modifier.fillMaxSize()) {
+        AsyncImage(
+            model = survey.coverImageUrl,
+            contentDescription = stringResource(id = R.string.survey_detail_background_image),
+            modifier = Modifier.fillMaxSize(),
+            placeholder = painterResource(id = R.drawable.ic_launcher_background),
+            contentScale = ContentScale.Crop
+        )
+        Image(
+            painter = painterResource(id = R.drawable.survey_overlay),
+            contentDescription = null,
+            modifier = Modifier.fillMaxSize(),
+            contentScale = ContentScale.Crop
+        )
+        HorizontalPager(
```

Comment:
Why are we using a pager here? Or is this preparation for showing the questions in a later PR? 🤔

Reply:
Yes I am planning to use the pager for each question